### DISCUSSION TITLE
How should I move folders around my project?

### DISCUSSION THREAD

**Discussion by HymanZHAN** [nvim-neo-tree/neo-tree.nvim#1536] [Q&A]
I tried different options but it's really confusing...

![image](https://github.com/user-attachments/assets/27f733ff-5583-4fd3-aa8f-726e3fd6d18f)

Let's say I want to move my `core` folder up a level so that it lives in the `src` directory. I hover my cursor on the `core` directory and hit `m`, what should I type in here?

Thanks in advance!

**Comment by miversen33** [comment]
If you want to move `core` up to source, you would enter `../core` for the path. `m` allows you to specify the new path you want the item to be found at. `../` is the relative path for "my parent".

**Reply by HymanZHAN** [reply]
Unfortunately, that doesn't work. 

![image](https://github.com/user-attachments/assets/e95d8b7f-5cfc-4c72-b075-33ed9629f4b0)

So that's where it gets confusing... When I try to `m`ove the folder, what is `pwd` referring to? According to this message, the (imaginary) `pwd` should be inside the `core` folder, right? So I think `../../core` should work. But if I type in `../../core` and hit enter, errors show up:

![image](https://github.com/user-attachments/assets/76943bd2-f368-458c-95da-2bc822fb6d39)

Something like: 
```
[Neo-tree ERROR] /SAPDevelop/Projects/ng-form-demo/src/app/core :  ENOENT: no such file or directory: /SAPDevelop/Projects/ng-form-demo/src/app/core
[Neo-tree ERROR] /SAPDevelop/Projects/ng-form-demo/src/app/core/../.. :  ENOENT: no such file or directory: /SAPDevelop/Projects/ng-form-demo/src/app/core/../..
```

The folder did get moved successfully though...

**Reply by miversen33** [reply]
If the file moved but you are seeing an error, it might be worth submitting a bug report

**Reply by imkgh** [reply]
Same here, I am running into the same issue, 🤣

**Reply by jeanersten** [reply]
im also running the same issue with cut and paste too

**Reply by arf20** [reply]
I still have the same issue
